genesyscloud/resource_genesyscloud_routing_skill_group.go

```diff
@@ -201,13 +208,123 @@ func createOrUpdateSkillGroups(ctx context.Context, d *schema.ResourceData, meta
 		log.Printf("Updated skill group %s", skillGroupsRequest.Name)
 	}
 
+	// Update member division IDs
+	memberDivisionIds := d.Get("member_division_ids").([]interface{})
```

Comment:
This overall function is getting pretty big.  Do we want to move the GroupMember logic to their own set of functions.